downstream/modules/platform/proc-settings-platform-gateway.adoc

```diff
@@ -0,0 +1,82 @@
+[id="proc-settings-platform-gateway"]
+
+= Platform gateway
+
+//To be added to Donna's AAP/UI document for 2.5 
+
+The Platform gateway is the service that handles authentication and authorization for {PlatformNameShort}. 
```

Comment:
```suggestion
The Platform gateway is the service that handles authentication and authorization for {PlatformNameShort}. 
```
Do we have an attribute for Platform gateway?

Reply:
No, there's one for the menu selection.